src/oaklib/utilities/lexical/lexical_indexer.py

```diff
@@ -265,6 +266,9 @@ def lexical_index_to_sssom(
     :param ensure_strict_prefixes: If true, prefixes & mappings in SSSOM MappingSetDataFrame will be filtred.
     :return: SSSOM MappingSetDataFrame object.
     """
+    if prefix_map and extended_prefix_map:
```

Comment:
## Exception handling
Throws error if both `prefix_map` and `extended_prefix_map` passed, as that creates ambiguity. The user should pick one.